packages/cli/test/functional/test_site/expected/markbind/js/pageVueRenderFns.min.js

```diff
@@ -0,0 +1 @@
"hljs-tag\"}},[_c('span',{pre:true,attrs:{\"class\":\"highlighted\"}},[_v(\"</\")]),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"foo\")]),_v(\">\")]),_v(\"\\n\")])])]),_c('p',[_c('strong',[_c('code',{pre:true,attrs:{\"class\":\"hljs inline no-lang\"}},[_v(\"highlight-lines\")]),_v(\" attr with line-part syntax should highlight only at specified substring\")])]),_v(\" \"),_c('pre',[_c('code',{pre:true,attrs:{\"class\":\"hljs xml\"}},[_c('span',[_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"<\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"foo\")]),_v(\">\")]),_v(\"\\n\")]),_c('span',[_v(\"  \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"<\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"bar\")]),_v(\" \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-attr highlighted\"}},[_v(\"type\")]),_v(\"=\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-string\"}},[_v(\"\\\"name\\\"\")]),_v(\">\")]),_v(\"goo\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"</\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"bar\")]),_v(\">\")]),_v(\"\\n\")]),_c('span',[_v(\"  \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"<\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name highlighted\"}},[_v(\"baz\")]),_v(\" \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-attr\"}},[_v(\"type\")]),_v(\"=\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-string\"}},[_v(\"\\\"name\\\"\")]),_v(\">\")]),_v(\"goo\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"</\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name highlighted\"}},[_v(\"baz\")]),_v(\">\")]),_v(\"\\n\")]),_c('span',[_v(\"  \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"<\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"qux\")]),_v(\" \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-attr\"}},[_v(\"type\")]),_v(\"=\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-string\"}},[_v(\"\\\"name\\\"\")]),_v(\">\")]),_c('span',{pre:true,attrs:{\"class\":\"highlighted\"}},[_v(\"go,o\")]),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag highlighted\"}},[_v(\"</\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"qux\")]),_v(\">\")]),_v(\"\\n\")]),_c('span',[_v(\"  \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"<\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"quux\")]),_v(\" \"),_c('span',{pre:true,attrs:{\"class\":\"hljs-attr\"}},[_v(\"type\")]),_v(\"=\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-string\"}},[_v(\"\\\"name\\\"\")]),_v(\">\")]),_c('span',{pre:true,attrs:{\"class\":\"highlighted\"}},[_v(\"go'o\")]),_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"</\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"quux\")]),_v(\">\")]),_v(\"\\n\")]),_c('span',[_c('span',{pre:true,attrs:{\"class\":\"hljs-tag\"}},[_v(\"</\"),_c('span',{pre:true,attrs:{\"class\":\"hljs-name\"}},[_v(\"foo\")]),_v(\">\")]),_v(\"\\n\")])])]),_c('p',[_c('strong',[_c('cod
```

Comment:
Thanks! Look forward to the finished product. @wxwxwxwx9 

quick comment: let's split this up - one file per page (or just inline it into the html) to make this scalable for something like the 2103 site.

Reply:
Thanks for the advice! I have split it accordingly. :-)

But I'm not sure if the naming convention that I came up with is the best. Let me know what you think about it!